curseforge-modpack-downloader.sh

```diff
@@ -21,36 +22,33 @@
 # TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
 # SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
-if [ $# -lt 1 ]; then
+# check arguments
+if [ -z "$1" ]; then
```

Comment:
Change `[  -z "$1" ]` to `[  $# != 1 ]` maybe better.
Because this program expects only 1 arg. What do you think about it?

Reply:
I agree, this makes more sense than either `[ $# -lt 1 ]` or `[ -z "$1" ]`, as this approach doesn't allow any undefined user input options. Didn't think of that. Also, the smaller overhead comparing `[ $# -lt 1]` is also an argument.